zerver/lib/send_email.py

```diff
@@ -124,6 +124,7 @@ def build_email(
         # work over time, even if the prod-static directory is cleaned
         # out; as such, they just use a STATIC_URL prefix.
         "email_images_base_url": settings.STATIC_URL + "images/emails",
+        "help_images_base_url": settings.STATIC_URL + "images/help",
```

Comment:
One reason why we have the `images/emails` directory is to remind ourselves that the URLs in that directory need to remain stable over time. We don’t want to force ourselves to make that commitment about `images/help`.

Perhaps we should copy or symlink this image into `images/emails` instead.

Reply:
That makes sense to me, thanks!